addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-json.html

```diff
@@ -38,19 +38,21 @@ <H4>About riskdesc</H4>
                             "method": "GET",
                             "param": "q",
                             "attack": "&lt;/font&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;font&gt;",
-                            "evidence": "&lt;/font&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;font&gt;"
+                            "evidence": "&lt;/font&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;font&gt;",
+                            "otherinfo": ""
                         },
                         {
                             "uri": "http://localhost:8080/bodgeit/contact.jsp",
                             "method": "POST",
                             "param": "comments",
                             "attack": "&lt;/td&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;td&gt;",
-                            "evidence": "&lt;/td&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;td&gt;"
+                            "evidence": "&lt;/td&gt;&lt;scrIpt&gt;alert(1);&lt;/scRipt&gt;&lt;td&gt;",
+                            "otherinfo": ""
+      
                         }
                     ],                   
                     "count": "2", 
                     "solution": "&lt;p&gt;Phase: Architecture and Design&lt;/p&gt;&lt;p&gt;Use a vetted library or framework that does not ...&lt;/p&gt;",
-                    "otherinfo": "",
```

Comment:
This should be restored. Although we know it isn't technically correct, removing it may break handling/processing that existing users have in place.

Reply:
okay I will correct that eveywhere.

Reply:
@thc202 that is what you meant right?

Reply:
Yes.

Reply:
This still needs to be addressed